apps/react-world/src/components/shared/banner/Banner.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from '@emotion/styled';
+
+export const Banner = styled.div`
+  color: #fff;
+  background: #333;
+  padding: 2rem;
+  margin-bottom: 2rem;
+`;
```

Comment:
현재 Banner 관련 속성이 이렇게 나뉘어져 있는데요,
- Banner.tsx
- BannerTitle.tsx

`apps/react-world/src/components/shared/banner/Banner.styled.ts` 로 파일을 하나 만들어서... 분리되어 있는 스타일 구문을 합치는 것이 좀 더 직관적일 것 같은데요, 우성님께서는 어떻게 생각하실까요?

```suggestion
// Banner.styled.ts
import styled from '@emotion/styled';

export const Banner = styled.div`
  color: #fff;
  background: #333;
  padding: 2rem;
  margin-bottom: 2rem;
`;

export const BannerTitle = styled.h1`
  text-shadow: 0px 1px 3px rgba(0, 0, 0, 0.3);
  margin-bottom: 0px;
`;
```

Reply:
`.styled.ts` 방식으로 스타일을 모아두는 형태도 가능하군요..! 스타일 코드가 많지 않아서 이 방향 좋은 것 같습니다 🙌

Reply:
@InSeong-So 말씀대로 한다면 banner 폴더 안에 기존 파일 두 개는 지워지고 스타일 파일 Banner.styled.ts(x?) 파일 하나만 남도록 하면 되는 것 맞을까요? 폴더 안에 컴포넌트 파일이 따로 없고 styled 파일 안에 styled component 선언만 있는게 원래 자연스러운 방식인가 헷갈려서요 ㅎ

Reply:
우선 위처럼 했습니다! 하면서 하나 추가로 들었던 궁금증은, HomeBanner 쪽도 컴포넌트 선언과 스타일이 같이 있는데 여기도 동일한 기준일까 싶어서 HomeBanner.tsx 와 HomeBanner.styled.ts 두 개로 분리해보았습니다.

Reply:
오 저도 파랑의 말씀에 동의해요~! HomeBanner 쪽도 동일한 기준으로 가져가는 것도 좋은 것 같아요!

저는 `.ts`/`.tsx`로 분리하는 기준은 그 파일 안에 JSX(Html 태그가 아닌 Element 태그 ex: `<HomeBannerTitle />` 등)이 있는지 여부로 판단해요!
styled 파일엔 스타일 선언부만 존재하기에 `.ts`로 사용되는 것이 맞는 것 같아요!

Reply:
> 오 저도 파랑의 말씀에 동의해요~! HomeBanner 쪽도 동일한 기준으로 가져가는 것도 좋은 것 같아요!
> 
> 저는 `.ts`/`.tsx`로 분리하는 기준은 그 파일 안에 JSX(Html 태그가 아닌 Element 태그 ex: `<HomeBannerTitle />` 등)이 있는지 여부로 판단해요! styled 파일엔 스타일 선언부만 존재하기에 `.ts`로 사용되는 것이 맞는 것 같아요!

오 자세한 설명 감사합니다! ㅎㅎ